src/main/java/be/seeseemelk/mockbukkit/scheduler/BukkitSchedulerMock.java

```diff
@@ -73,6 +73,7 @@ public void performOneTick()
 				}
 				else
 				{
+					runners.put(task.getTaskId(),task);
```

Comment:
little formatting remark
```suggestion
					runners.put(task.getTaskId(), task);
```